notes/bitcoin-as-anti-authoritarian.md

```diff
@@ -42,6 +56,21 @@ This is best evidenced by the Canadian convoys in 2022 which attempted to take i
 
 The complete ban of [crypto assets](../concepts/cryptoasset.md) by the People's Republic of China also does not lend credibility to the thesis that [crypto assets](../concepts/cryptoasset.md) are outside the remit of authoritarian controls and their restriction on capital movement and controls over domestic [money services business](../concepts/money-services-business.md).
 
+### Subclaim 2: An unregulated transnational payment rail is desirable
+
+From the perspective of civil society a global supranational payment system which is [censorship resistant](../concepts/censorship-resistence.md) against nation state actors **is not desirable**. The world already struggles with an excess of offshore tax evasion and dark money flows, as evidenced by the recent Panama Paper leaks.
+
+#### Crypto facilitates tax evasion
```

Comment:
I think point here is more: **If** crypto useful for dissidents then it is also useful for X, Y, Z.

so you have to be a bit subtle here:

Evaluation 1: no it isn't actually good because it is trackable ...

Evaluation 2: ... and if it were good (e.g. tech got better) the problem is that it would also be good for all these bad things.

Ultimately the marginal benefit to dissidents would be outweighed by facilitating money laundering etc.